chapters/2.md

```diff
@@ -1,4 +1,195 @@
 # Chapter 2
 
-## Getting Started
+# Getting Started
+The last chapter focused on WHY to use Monero; in this chapter you will learn HOW to use Monero. You can master using Monero without needing to learn any of the complex cryptographic or network technical details, so that extra information is saved for later in the book. **Chapter 2 will cover all the practical skills you’ll need to get started receiving,
```

Comment:
> You can master using Monero without needing to learn any of the complex cryptographic or network technical details, so that extra information is saved for later in the book.

`Using Monero is actually quite simple; you can master using Monero without needing to learn anything about the details of the cryptography or the way the network works, but we will be covering those topics later in later chapters.`

> **Chapter 2 will cover all the practical skills you’ll need to get started receiving, storing, and spending your Moneroj**.

*newline in middle of sentence*